web/content/docs/benchmarks/hydro-component/theis/HC_Theis.pandoc

```diff
@@ -0,0 +1,65 @@
++++
+author = "Marc Walther"
+weight = 142
+project = "Parabolic/ComponentTransport/Theis/HC_theis.prj"
+date = "2019-05-02T14:41:09+01:00"
+title = "Theis solution for well pumping"
+
+[menu]
+  [menu.benchmarks]
+    parent = "hydro-component"
+
++++
+
+{{< data-link >}}
+
+
+## Overview
+
+Abstraction of water from an aquifer is a common procedure, as groundwater is usually better protected from contamination and steadily available even in dry seasons. For a sustainable water usage, the evaluation of the spatio-temporal behaviour of the water level for a specified abstraction rate is of high importance for water management. 
+
+This benchmark verifies the groundwater level drawdown for an aquifer that is subjected to pumping.
+
+
+## Problem description
+
+For pumping from a well, Theis (1935) proposed an analytical solution to calculate the water level drawdown over time and distance from the well. A short summary of the solution can be seen in 
+web/content/docs/benchmarks/liquid-flow/liquid-flow-theis-problem.pandoc
+
+Here, we verify pumping abstraction with Theis for the `ComponentTransport` process using a 3D setup. The setup is adapted from Walther (2014), section 3.2.
+
+
+### Model setup
+
+The setup comprises a 1/8th slice of a full circle (see figure 1). 
+
+{{< img src="../BCs.png" title="Mesh and boundary conditions (BC); blue = outer Dirichlet pressure and concentration BC, red = inner Neumann abstraction BC.">}}
+
+The outer boundary condition is set as Dirichlet with a hydrostatic pressure along the shell surface of the slice equivalent to a head of $h = 0 m$ (i.e. water level equals top of domain). For mass transport, a Dirichlet boundary conditions with concentration $c = 0$ is set at the outer shell. The inner boundary condition is equivalent to the eighth of a total abstraction rate of $Q_t = 15 m^3/d$ for a full cylinder. *NB: In the `ComponentTransport` process, the Neumann BC is given as mass flux and has to be calculated per area, such that the value for the project file is $Q = Q_t / 8 / A \cdot \rho = 2.83542E-03 m^3/s/m^2 \cdot kg/m^3$ with fluid density $\rho = 1000 kg/m^3$ and abstraction area $A = 7.65 m^2$.* 
```

Comment:
The units within this section are a bit hard to read. Maybe one could use m^3/s/m^2 \cdot kg/m^3 = \frac{kg}{s m^2} which directly hints at mass flux.